Assignment1/styles.scss

```diff
@@ -0,0 +1,342 @@
+$font-stack: Poppins, sans-serif;
```

Comment:
Why do you have to styles files?

Reply:
Not sure I understand the question. Did you mean why there are 2 styles?

Reply:
Why do you have two styles files?*

Reply:
Because I used Sass preprocessor, and when I process the .scss file, the .css file is created, which is the one used linked in the html. At least that is how I understood sass works.